### ISSUE TITLE
Can't log in to the website

### ISSUE THREAD

**Issue by Glothox** [snazy2000/Ninjah-Gaming#3]
Whenever I seem to try to log in to the website I get asked to verify my account and to verify it through e-mail. Since I haven't gotten an E-mail I click on the "resend verification" button and enter in the credentials. Appearently, my E-mail is not a valid e-mail (jan.egilodegard@yahoo.no) and I've also tried with 2 other E-mails without success.. Also, according to the website, the verification key is also wrong (the one below the E-mail credential entry)

Basically, can't log into Ninjah-gaming due to me having to verify my account, something I am not able to do.

**Comment by Sladewill** [comment]
I'll take a look into it when I get home, this maybe because of a bug on the old website where some users managed to get on without verifying their accounts and the verification may not be working as it was rushed yesterday to try and get the website live.

**Comment by snazy2000** [comment]
I have activated your account for now :)

**Comment by snazy2000** [comment]
Fixed
